malloc.h

```diff
@@ -6,9 +6,30 @@ typedef void (*rtr_free_t)(void *mem);
 typedef void *(*rtr_calloc_t)(size_t nmemb, size_t size);
 typedef void *(*rtr_realloc_t)();
 
+typedef void *(*rtr_mmap_t)(void *addr, size_t length, int prot, int flags,
+	int fd, off_t offset);
+typedef int (*rtr_munmap_t)(void *addr, size_t length);
+
+#ifndef __APPLE__
+
+typedef int (*rtr_brk_t)(void *addr);
+typedef void *(*rtr_sbrk_t)(intptr_t increment);
+
+#endif
+
 RETRACE_DECL(malloc);
 RETRACE_DECL(free);
 RETRACE_DECL(calloc);
 RETRACE_DECL(realloc);
 
+RETRACE_DECL(mmap);
+RETRACE_DECL(munmap);
+
+#ifndef __APPLE__
```

Comment:
@jjr840430 could you help add a comment there that they are deprecated on macOS, and this line is necessary to pass? Thanks!